### PR TITLE
feat: add OSCR tooltip to user page

### LINKED ISSUE #3854: Feature: Describe OSCR rating
### Suggested solution

e.g.  at
Screenshot 2024-08-05 at 12 19 17 PM

I see an OSCR rating but have no clue what the scale is or what it means. A hover tooltip with more information would be nice, as well as a link to more information.

### PR DESCRIPTION
**Pull request by zeucapua**
## Description

Implements an `InfoTooltip` to the user page (`/u/[username]`) to explain what OSCR is.

## Related Tickets & Documents

Closes #3854 

## Steps to QA

1. Go to any user page

## Tier (staff will fill in)

- [ ] Tier 1
- [ ] Tier 2
- [ ] Tier 3
- [x] Tier 4

### CODE REVIEW

**Review by nickytonline — COMMENTED** [review]
The info tooltip works, but it's super small. It's hard to tell what it is as it looks like a circle with a small dot in it.

CleanShot 2024-08-05 at 14 36 46

File: components/organisms/ContributorProfileTab/contributor-profile-tab.tsx [hunk 1]
```diff
@@ -371,8 +371,8 @@ const ContributorProfileTab = ({
             </div>
             <div className="grid grid-cols-2 2xl:grid-cols-4 justify-between gap-2 lg:flex-row md:gap-12 lg:gap-16">
               <div>
-                <span className="flex gap-2 text-xs text-light-slate-11">
-                  OSCR Rating
+                <span className="flex gap-2 text-sm">
```
Comment (nickytonline) [inline components/organisms/ContributorProfileTab/contributor-profile-tab.tsx:374]: The info icon is larger now, but the text ends up being misaligned now.

CleanShot 2024-08-05 at 15 20 25

CleanShot 2024-08-05 at 15 20 06

**Review by nickytonline — CHANGES_REQUESTED** [review]

**Review by jpmcb — APPROVED** [review]
Copy looks great!

**Review by brandonroberts — APPROVED** [review]

File: components/organisms/ContributorProfileTab/contributor-profile-tab.tsx (hunk 1 above)
Comment (nickytonline) [reply components/organisms/ContributorProfileTab/contributor-profile-tab.tsx:374]: It's still off, but I figured a way to have it bigger and line up properly.

For the OSCR set the JSX to:

```jsx
              <div>
                <span className="relative text-xs text-light-slate-11 flex gap-0.5 items-center">
                  <span>OSCR Rating</span>
                  <span className="text-sm grid place-content-center">
                    <InfoTooltip information="OSCR evaluates the engagement and impact of contributors across the entire open source ecosystem." />
                  </span>
                </span>
                <div className="flex mt-1 text-lg md:text-xl lg:text-2xl !text-black leading-none">
                  <OscrButton rating={contributor?.oscr} hideRating={!Boolean(user)} />
                </div>
              </div>
```

And for the other stats, add `flex-items center`, e.g.

```diff
              <div>
-                <span className="text-xs text-light-slate-11">PRs opened</span>
+                <span className="text-xs text-light-slate-11 flex items-center">PRs opened</span>

                {totalPrs ? (
                  <div className="flex mt-1 !text-lg md:!text-xl lg:!text-2xl !text-black !leading-none">
                    {totalPrs} PRs
                  </div>
                ) : (
                  <div className="mt-1">{DATA_FALLBACK_VALUE}</div>
                )}
              </div>
```

**Review by nickytonline — CHANGES_REQUESTED** [review]

**Review by nickytonline — APPROVED** [review]
Approving after chatting about it in our 1:1 that you'll be making the suggested changes.

**Pull request merged**